custom/icds_reports/static/js/angular-services/locations.service.js

```diff
@@ -1,6 +1,17 @@
 window.angular.module('icdsApp').factory('locationsService', ['$http', '$location', function($http, $location) {
     var url = hqImport('hqwebapp/js/initial_page_data').reverse;
     var gtag = hqImport('analytix/js/google').track;
+
+    function transformLocationTypeName(locationTypeName) {
```

Comment:
just moving code I wanted to reuse. though this commit could affect web dashboard so should be reviewed carefully (I did test extensively).

Reply:
Also confirming I grepped the whole codebase for calls to these two functions